### PR TITLE
SIMSBIOHUB-25: Admin dashboard and  project roll up

### PR DESCRIPTION
**Pull request by al-rosenthal**
# Overview

## Links to Jira tickets

[SIMSBIOHUB-25](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-25)

## Description of relevant changes

- Added admin dashboard to biohub
- Added new endpoint to serve data to admin dashboard
- Updated routing to account for admin only access
- Migration: updated source transform to include new fields
- Migration: added new columns for tracking submitter system and elastic search object

## Testing Procedure
1. Get both sims and biohub running locally
2. Open biohub and login
3. You should be redirected to the new dashboard page
4. See there are `No Pending Security Reviews`
5. Create a project and survey in SIMS
6. Add documents to both project and survey (track the number of files in total)
7. Refresh the dashboard page (this might take a bit as the transform action is queued)
8. See the data table is updated with the project you just created. Compare the Files to Review with the total files you added to both Project and Survey (they should be the same)
9. Click the name of the project and be redirected to the project details page

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

### CODE REVIEW

File: database/src/migrations/20230518125700_update_record_sims_eml_transformation.ts [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+      FROM
+        project_objects
+    ),
+    related_project_objects_arr AS (
+      SELECT
+        jsonb_agg(project_object) project_objects_arr
+      FROM
+        related_project_objects
+    ),
+    project_type AS (
```
Comment (al-rosenthal) [inline database/src/migrations/20230518125700_update_record_sims_eml_transformation.ts:197]: this is the only change to the transformation code. Adding new data for the submission metadata table, it is being generated here and added to the transform on line 225

File: api/src/paths/administrative/review/list.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+const defaultLog = getLogger('paths/administrative/review/list');
+
+export const GET: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN],
+          discriminator: 'SystemRole'
```
Comment (NickPhura) [inline api/src/paths/administrative/review/list.ts]: nit: I think data admins will need access to

Comment (al-rosenthal) [reply api/src/paths/administrative/review/list.ts]: good call, I updated the route guards to include data admins as well

**Review by anissa-agahchen — DISMISSED** [review]
I pulled the code and tested in manually ... works as expected

**Review by anissa-agahchen — APPROVED** [review]

**Review by NickPhura — APPROVED** [review]
🥝

**Pull request merged**